packages/drivers/odsp-driver/src/test/odspSummaryUploadManagerTests.spec.ts

```diff
@@ -31,6 +31,7 @@ describe("Odsp Summary Upload Manager Tests", () => {
             async (options: TokenFetchOptions, name?: string) => "token",
             logger,
             epochTracker,
+            { blobDeduping: true },
```

Comment:
Would be great to test with and without (i.e. shipping code as well :))

Reply:
added